__tests__/bitbucket_service_test.spec.ts

```diff
@@ -38,4 +39,23 @@ describe('Bitbucket Service Tests', () => {
         done(err);
       });
   });
+
+  it('should return exposed container port', (done: any) => {
+    const gr = new GitSource(
+      "https://bitbucket.org/akashshinde123/tutorial-react-docker",
+      SecretType.NO_AUTH,
+      null
+    );
+
+    const gs = new BitbucketService(gr);
+    gs.getDockerfileContent()
+      .then((content: string) => {
+        const parser = new DockerFileParser(content);
+        const port = parser.getContainerPort();
+        expect(port).toEqual(5000);
```

Comment:
@akashshinde shouldn't there be negative test scenario as well e.g.
- to check if a proper error is returned when Dockerfile is not present

Reply:
Added negative tests https://github.com/redhat-developer/git-service/pull/5/commits/c8bef3d1d7961e40d37501b1e9ad914ec6f8c75a